dotnet/src/Microsoft.AutoGen.Agents.Runtime/Host.cs

```diff
@@ -7,6 +8,7 @@ public static class Host
     public static async Task<WebApplication> StartAsync(bool local = false)
     {
         var builder = WebApplication.CreateBuilder();
+        builder.AddServiceDefaults();
```

Comment:
Are we always expecting the runtime host to run in a seperete process?

Reply:
no, its explicitly in the same process (different thread and WebApplication) if local = true here.